data/json/items/comestibles/raw_veggy.json

```
@@ -72,7 +73,6 @@
     "stim": 1,
     "spoils_in": "20 days",
     "description": "This aromatic rhizome can be eaten as is in an emergency, but it is much better used for making beverages.  Its taste is pleasant but it is also tough and stringy.",
-    "comestible_type": "FOOD",
```

```suggestion
    "comestible_type": "FOOD",
```
Moving this around is unecessary